codebuild/spec/buildspec_fuzz.yml

```diff
@@ -60,7 +60,7 @@ phases:
     commands:
       - |
         cmake . -Bbuild \
-        -DCMAKE_PREFIX_PATH=$LIBCRYPTO_ROOT \
+        -DCMAKE_PREFIX_PATH=/usr/local/$S2N_LIBCRYPTO \
```

Comment:
🤔 Why do you need the "/usr/local" path? Does it not find awslc if you don't use it?

Reply:
It will not find the libcrypto and here is a test run without the /usr/local/ path: [Link to CodeBuild](https://us-west-2.console.aws.amazon.com/codesuite/codebuild/024603541914/projects/s2nFuzzBatchScheduled/build/s2nFuzzBatchScheduled%3Aba7fc30b-0c94-4fc5-a92e-30afa272e9d0/log?region=us-west-2)
I was following how buildspec_sanitizer.yml sets it up https://github.com/aws/s2n-tls/blob/341a69ed6fc4fa8f51c09b9cc477223cec4d8e60/codebuild/spec/buildspec_sanitizer.yml#L106